x-pack/platform/plugins/shared/alerting/server/routes/rule/apis/update/transforms/transform_update_body/v1.test.ts

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
@justinkambic I am wondering why is this test added as part of the front end PR? Could it be this is a leftover or something from the backend PR?